### PR TITLE
RELATED: BB-1850 measure value filter without toggle button

### PR DESCRIPTION
**Pull request by strouhaljan**
Making the developer experience better. Removed the ability to replace the default dropdown filter button by custom one. Now you can use whichever button you like, but you need to take care of dropdown visibility state on your own. I found it's a better approach as it's done similarly in many UI component libraries (Material UI etc.).

Needs documentation update as well: https://github.com/gooddata/gooddata-ui/pull/252

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5362/ : SUCCESS in 14m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/849/ : SUCCESS in 10s

File: examples/src/components/MeasureValueFilterDropdownExample.jsx [hunk 1]
```diff
@@ -22,35 +24,75 @@ const measures = [franchiseFeesMeasure, franchiseSalesMeasure];
...
+
+DropdownButton.propTypes = {
+    isActive: PropTypes.bool.isRequired,
+    measureTitle: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+};
+
+export class MeasureValueFilterDropdownExample extends React.PureComponent {
+    state = {
+        filter: [],
```
Comment (xMort) [inline examples/src/components/MeasureValueFilterDropdownExample.jsx]: I would call it `filters`. I know that there will be just one but it is an array and it goes to `filters` property of the pivot component.

File: examples/src/components/MeasureValueFilterDropdownExample.jsx [hunk 2]
```diff
@@ -22,35 +24,75 @@ const measures = [franchiseFeesMeasure, franchiseSalesMeasure];
...
+                <DropdownButton
+                    onClick={this.toggleDropdown}
+                    isActive={displayDropdown}
+                    measureTitle="Measure"
                 />
+                {displayDropdown ? (
+                    <MeasureValueFilterDropdown
+                        onApply={this.onApply}
+                        onCancel={this.onCancel}
+                        measureIdentifier={franchiseSalesMeasure.measure.localIdentifier}
```
Comment (xMort) [inline examples/src/components/MeasureValueFilterDropdownExample.jsx]: Would it be hard to rename this to `measureLocalIdentifier`? Measure identifier could be a metadata measure identifier and some people could be confused. What do you think about it?

Comment (xMort) [reply examples/src/components/MeasureValueFilterDropdownExample.jsx]: It seems that we are using this term in this meaning in multiple other places so let us continue with this unfortunate trend.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5375/ : SUCCESS in 14m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/866/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5395/ : SUCCESS in 18m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/893/ : SUCCESS in 17s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5414/ : FAILURE in 1m 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/910/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5415/ : SUCCESS in 4m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/911/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build failed (check-teardown pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-stop-zuul-docker/824/ : FAILURE in 4m 48s

**Pull request closed without merging**